src/backend/export/outputVendors/googleSheets/googleAuth.ts

```diff
@@ -8,8 +8,7 @@ export const clientId = GOOGLE_CLIENT_ID;
 export const clientSecret = GOOGLE_CLIENT_SECRET;
 export const redirectUri = homepage;
 export const scopes = [
-  'https://www.googleapis.com/auth/drive.metadata.readonly',
-  'https://www.googleapis.com/auth/spreadsheets',
+  'https://www.googleapis.com/auth/drive.file'
```

Comment:
You talked about you changed the scopes, and I thought you changed the technique.
Can you explain (here or on Whatsapp) what is the problem, you solved?
Because I thought we need to declare the tightest scopes.

Reply:
Google classifies different scopes by how sensitive they are. If they are considered sensitive they give the user a warning and require manual approval of the app by google. This scope for example: `...auth/spreadsheets` gives the app read and write access to all of the user's spreadsheets. That is way more than we need and they consider this sensitive.

Instead in this implementation I use only one scope - `...auth/drive.file` and this scope allows the app access to the spreadsheets that it created. This a much less sensitive permission and so it doesn't show the warning and doesn't require google to review and approve the app.

![Screen Shot 2020-12-22 at 15 13 07](https://user-images.githubusercontent.com/7272927/102892370-6f541100-4468-11eb-9a77-505769e1a68d.png)

Reply:
Cool